app/models/grda_warehouse/auth_policies/hud_report_policy.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+class GrdaWarehouse::AuthPolicies::HudReportPolicy < GrdaWarehouse::AuthPolicies::BasePolicy
```

Comment:
breaking ground on a policy class for hud reports. This uses the project_ids to get the permissions, not sure if that's exactly right